CHANGELOG

```diff
@@ -436,7 +436,7 @@ Version 3.4.1 (2015-07-06)
 	Fix spelling in accepted connection notification email (strk)
 	Show image size warning in a human readable format (rabuzarus)
 	Move ACL window in template (rabuzarus)
-	New option "-s" in util/run_xgettext.sh (fabrixxm)
```

Comment:
I don't think this should be changed in the historic documentation of the CHANGELOG as there will be a new entry with the correct release the the binaries were moved.

Reply:
Reverted.